Documentation/specs/runtime-configuration-file.md

```diff
@@ -25,8 +25,15 @@ The files are both JSON files stored in UTF-8 encoding. Below are sample files.
 ```json
 {
     "runtimeOptions": {
-        "gcServer": true,
-        "gcConcurrent": false,
+
+        "configProperties": {
+            "System.GC.Server": true,
```

Comment:
I guess the idea is these might eventually be exposed as static properties for managed code to use, and namespacing is a good idea anyway so an app property doesn't interfere with runtime properties.